packages/paste-core/components/breadcrumb/__tests__/breadcrumb.spec.tsx

```diff
@@ -45,7 +45,7 @@ describe('Breadcrumb', () => {
         <BreadcrumbItem href="#">foo</BreadcrumbItem>
       </Breadcrumb>
     );
-    const renderedAnchor = screen.findByRole('link');
+    const renderedAnchor = screen.getAllByRole('link');
```

Comment:
`findByRole` is asynchronous. This is likely the main culprit of the `createEvent` error on react 17 tests.

`getBy` is not, and arguably does a better job.

Reply:
🪙 🪙 The find methods are useful for testing, but without an await statement will cause async errors. [KCD summarizes it here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find) succinctly.

I wish `.find...` methods threw a better error when used without some kind of async logic.